<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningHintDialog</class>
 <widget class="QDialog" name="WarningHintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>警告提示信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="stopCompressor1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>1#空压机屏蔽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stopCompressor2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>2#空压机屏蔽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stopCompressor3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>3#空压机屏蔽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stopDryer1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>1#冷干机屏蔽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stopDryer2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>2#冷干机屏蔽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="stopDryer3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>3#冷干机屏蔽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
